lib/account-streamer.ts

```diff
@@ -1,9 +1,9 @@
 import WebSocket from 'isomorphic-ws'
 import _ from 'lodash'
-import type { JsonMap, JsonValue } from './utils/json-util'
-import { JsonBuilder } from './utils/json-util'
-import TastytradeSession from './models/tastytrade-session'
-import { MinTlsVersion } from './utils/constants'
+import type { JsonMap, JsonValue } from './utils/json-util.js'
```

Comment:
Can you revert all the `.js` suffixes? This is a typescript project. The javascript files should all get compiled into the `dist` folder.

Reply:
TS compiler does not rewrite import paths. This is something the TS team has been adamant about. Omitting `.js` extension here will result in an error for all node.js users with ERR_MODULE_NOT_FOUND for the compiled files in dist. See this stack overflow post for more details on this topic - https://stackoverflow.com/a/62626938

Ideally, we would want to adopt a bundler to bundle all the files in dist if we wish to keep extension less imports in src. This however is much larger change!

Reply:
Can you add some steps to reproduce the issue you are describing?

Reply:
@osdevisnot I see the issue. I'll add a couple things to the tsconfig and we can merge this.